### DISCUSSION TITLE
测试版不要取消端口号和代理全部啊

### DISCUSSION THREAD

**Discussion by konaisland** [getlantern/lantern-forum-cn#670] [General]
这两个功能很重要。

**Comment by MaxMKMax** [comment]
@konaisland 您好. 请您提供截图给我. 我帮你们反馈. 谢谢!

**Reply by konaisland** [reply]
<img width="586" alt="port" src="https://github.com/getlantern/lantern-forum-cn/assets/44970641/a3761e60-d308-4629-9ece-49235d7f2e27">

现在github上的，preview测试版 ，把这几个选项都取消了，不要啊！ 很好用的功能。

**Comment by z2wenfa** [comment]
表示赞同，现在只能全局代理和智能路由，但是智能路由有些外网网站无法使用，很多软件也有配置代理端口的需求，比如Claude code。

**Reply by IvanYe86** [reply]
谢谢，看到你这条我才找到网站一直刷不进去的原因
